docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -335,3 +335,22 @@ Only available in Grafana v7.1+.
 
 Click on a time series in the panel to see a context menu with a link to View in Metrics Explorer in Google Cloud Console. Clicking that link opens the Metrics Explorer in the Google Cloud Console and runs the query from the Grafana panel there.
 The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the Metrics Explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
+
+## Out-of-the-box dashboards
+
+> Only available in Grafana v7.3+.
```

Comment:
Please do not merge this until the 7.3 beta. @daniellee , FYI.

Reply:
Why? Have we changed this approach? Not optimal having PR's ready, but not merged for some time and then they're going to be merged for the beta. We want to merge as fast possible to make sure it works as expected in master. Guess the approach would be to open separate PR's for documentation then?

Reply:
Literally having a meeting today to discuss it, yes. And I'm just talking about docs, not the feature.

Which is not ideal, I know. I would like the docs to stay in lockstep with the code, just as this PR has it, but right now we are trying to figure out how to keep the future-state stuff out of the current docs as much as possible (currently 7.2 docs) without driving us all insane.

Reply:
Until we have solved https://github.com/grafana/grafana/issues/27384 we are stuck with version notices (which can be good even post that)

Reply:
@papagian which means we can merge this before 7.3 beta leaving the version notes intact.